### DISCUSSION TITLE
Cannot clone Bitbucket Cloud repo

### DISCUSSION THREAD

**Discussion by wtpalexander** [thebaselab/codeapp#472] [Q&A] [answered]
I’m getting “Too many auth redirects” when trying to clone a Bitbucket cloud repo. This used to work with my credentials. Do I need to use a special credential such as an app password? I’ve given it ago but sadly didn’t get a better result. 

Any ideas?

**Comment by bummoblizard** [comment, marked as answer]
Hi. It sounds like that the personal access token has expired. Generating a new one on Bitbucket should work.

**Reply by wtpalexander** [reply]
Hey, from what I can tell - Bitbucket Cloud doesn’t have personal access tokens. I’ve tried the “App Passwords” feature which appears, to me, to be the replacement equivalent but that doesn’t seem to work either. :(

**Reply by bummoblizard** [reply]
> Hey, from what I can tell - Bitbucket Cloud doesn’t have personal access tokens. I’ve tried the “App Passwords” feature which appears, to me, to be the replacement equivalent but that doesn’t seem to work either. :(

I see. Please also make sure you gave sufficient permission to the app password and use your Bitbucket username instead of email. Regardless I will try to create a Bitbucket account to test this.

**Reply by wtpalexander** [reply]
Ahhhh tried with just the user name and its working - thanks a bunch! :)

**Reply by bummoblizard** [reply]
> Ahhhh tried with just the user name and its working - thanks a bunch! :)

Great!
